enterprise/ha/src/test/java/org/neo4j/kernel/ha/cluster/TerminationOfSlavesDuringPullUpdatesTest.java

```diff
@@ -158,6 +161,13 @@ public void slavesDontTerminateAndReadConsistentDataWhenApplyingBatchSmallerThan
         race.go();
     }
 
+    private ClusterManager.ManagedCluster startCluster() throws Exception
+    {
+        ClusterManager.ManagedCluster cluster = clusterRule.startCluster();
+        cluster.await( allSeesAllAsAvailable() );
```

Comment:
this availability check is defined in clusterRule by default, we do not need it here